gwt/pom.xml

```diff
@@ -249,8 +249,7 @@
     </dependency>
     <dependency>
       <groupId>org.slf4j</groupId>
-      <artifactId>slf4j-jdk14</artifactId>
-      <version>1.7.7</version>
+      <artifactId>slf4j-api</artifactId>
```

Comment:
Did you check if it is really needed? 98% of this kind of dependencies I've found weren't...

Reply:
is necessary